chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/loadbalancer/JLoadBalancer.java

```diff
@@ -66,7 +67,9 @@ public QueryPoolLoadBalancer build() {
             switch (loadBalancer) {
                 case ONE_BY_ONE:
                     if (seed == null) {
-                        return new OneByOneLoadBalancer();
+                        SimpleCircularLoadBalancer loadBalancer = new SimpleCircularLoadBalancer();
+                        loadBalancer.setPairSupplierFactory(new OneByOnePairSupplierFactory());
```

Comment:
these changes are in the codebase already - https://github.com/griddynamics/jagger/blob/c4e77cfb5267fd478e6f8daef9e08f72eb2e94d4/chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/loadbalancer/JLoadBalancer.java